portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +27,16 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function myFunction(imgs) {
+  // Get the expanded image
+  var expandImg = document.getElementById("expandedImg");
```

Comment:
This could be "expandedImage", maybe? If you say "expand", it sounds like it's in the imperative form.